Error installing `ignite-cli` version `7.10.3`

Attempting to install `ignite-cli` version `7.10.3` throws the following error:

```bash
> npm i -g ignite-cli
npm ERR! code 1
npm ERR! path C:\Users\User\scoop\persist\nodejs-lts\bin\node_modules\ignite-cli
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c solidarity
npm ERR! 'solidarity' is not recognized as an internal or external command,
npm ERR! operable program or batch file.
```

---

Steps to reproduce

1. Run: `npm i -g ignite-cli`

---

The error indicates that `solidarity` is not installed but note that even if `solidarity` is installed, `ignite-cli` is still unable to be installed as the following error is thrown:

```bash
> npm i -g ignite-cli
npm ERR! code 3
npm ERR! path C:\Users\User\scoop\persist\nodejs-lts\bin\node_modules\ignite-cli
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c solidarity
npm ERR! No Solidarity Settings Found
npm ERR! Make sure you are in the correct folder or run solidarity onboard to create a .solidarity file for this project.
```

I'm getting this same issue on MacOS.  First I tried with `npx`, which failed silently.  
Then I installed globally and got the following error:
```bash
> npm install -g ignite-cli
npm ERR! code 127
npm ERR! path ***/.nvm/versions/node/v17.4.0/lib/node_modules/ignite-cli
npm ERR! command failed
npm ERR! command sh -c solidarity
npm ERR! sh: solidarity: command not found
```

I have also tried installing `solidarity` globally.  With that I get the following:
```bash
> npm install -g ignite-cli
npm ERR! code 3
npm ERR! path ***/.nvm/versions/node/v17.4.0/lib/node_modules/ignite-cli
npm ERR! command failed
npm ERR! command sh -c solidarity
npm ERR! No Solidarity Settings Found
npm ERR! Make sure you are in the correct folder or run solidarity onboard to create a .solidarity file for this project.
```

I've tried with npm 6/7 and node 14/15/16/17.

--- a/package.json
+++ b/package.json
@@ -38,8 +38,7 @@
     "ci:test": "yarn build && yarn ci:lint && jest --runInBand && yarn clean",
     "ci:publish": "yarn build && yarn semantic-release && yarn clean",
     "semantic-release": "semantic-release",
-    "clean": "rm -rf ./build && rm ./boilerplate/.gitignore.template",
-    "postinstall": "solidarity"
+    "clean": "rm -rf ./build && rm ./boilerplate/.gitignore.template"
   },
   "devEngines": {
     "node": ">=7.x",

